core/src/main/java/org/aerogear/mobile/core/metrics/MetricsPublisherListener.java

```diff
@@ -0,0 +1,11 @@
+package org.aerogear.mobile.core.metrics;
+
+
+import org.aerogear.mobile.core.http.HttpResponse;
+
+public interface MetricsPublisherListener {
+
+    void onPublishMetricsSuccess(HttpResponse response);
```

Comment:
Why do we need to expose response if it was a success?

Reply:
No need really, only for the test it would be more useful.